unlock-js/src/web3Service.js

```diff
@@ -134,18 +134,59 @@ export default class Web3Service extends UnlockService {
     }
   }
 
+  /**
+   * Method which returns the create2 address based on
+   * @private
+   */
+  _create2Address(unlockAddress, templateAddress, account, lockSalt) {
+    const saltHex = `${account}${lockSalt}`
+    const byteCode = `0x3d602d80600a3d3981f3363d3d373d3d3d363d73${templateAddress.replace(
```

Comment:
What is `0x3d602d80600a3d3981f3363d3d373d3d3d363d73`?

Reply:
Ha, these comments have disappeared :/ Will add them back.